lib/globus_client/identity.rb

```diff
@@ -43,12 +48,8 @@ def lookup_identity
 
     def extract_id(data)
       identities = data["identities"]
-      # Select identity with "used" or "private" status
-      matching_users = identities.select { |id| id["username"] == @email }
-      active_users = matching_users.select { |user| (user["status"] == "used" || user["status"] == "private") }
-      raise "No matching active Globus user found for #{@email}." if active_users.empty?
-
-      active_users.first["id"]
+      matching_user = identities.select { |id| id["username"] == @email }
```

Comment:
If we only care about the first match, we could consider `#find` instead of `#select` here, and dump the `#first`s below. We'd just need to be deal with the fact that `matching_user` can be `nil`? Or maybe that can't happen? (I don't think we can deal with `nil`s now anyway?)

Reply:
I like that approach to simplifying, along with the other suggestion you made about this method (returning the whole response, which is small.) Made that change, just had to adjust to string keys since no longer creating a new hash with symbol keys. 

We can't get `nil` match, since the way our globus is set up, just searching on a user with a stanford.edu email auto-provisions one. (So seems really unlikely?)

Reply:
Sounds good! Thx